src/rules/value-no-redundant-longhand/README.md

```diff
@@ -0,0 +1,58 @@
+# value-no-redundant-longhand
+
+Disallow redundant values in longhand properties.
```

Comment:
After the discussion below, this should be:

"Disallow redundant values in shorthand properties."